docs/howto/flask-async.rst

```diff
@@ -0,0 +1,48 @@
+======================================================
+How to write a Kubernetes charm for an Async Flask app
+======================================================
+
+In this how to we will configure 12 Factor Flask application
+to use asynchronous Gunicorn workers to be able to serve
+to multiple users easily.
+
+Make the rock async
+===================
+
+Before packing the rock make sure to put the following in ``requirements.txt``
+file:
+
+.. literalinclude:: code/flask-async/requirements.txt
+
+Configure the async application
+===============================
+
+Now let's enable async Gunicorn workers using a configuration option. We will
+expect this configuration option to be available in the Flask app configuration
+under the keyword ``webserver-worker-class``. Verify that the new configuration
+has been added using
+``juju config flask-async-app | grep -A 6 webserver-worker-class:`` which should
+show the configuration option.
+
+.. note::
+
+    The ``grep`` command extracts a portion of the configuration to make
+    it easier to check whether the configuration option has been added.
```

Comment:
Nitpick: I think you can remove this note. I think it makes more sense in a tutorial versus a how-to guide.